### DISCUSSION TITLE
💭 Decisão: Quando localizar um termo é uma boa ou uma má decisão?

### DISCUSSION THREAD

**Discussion by contraexemplo** [infuse-l10n-pt-br/Infuse#7] [Glossário]
O português brasileiro é uma língua que abraça muito termos emprestados de outros idiomas, e muitas áreas da tecnologia abraçam menções a termos técnicos no idioma original. Um exemplo é o da discussão #6: muitos eventos brasileiros usam a sigla CTF ou o termo Capture the Flag sem nenhuma tradução.

Logo, precisamos deliberar:
1. _Quando_ devemos traduzir termos técnicos?
2. Devemos inserir uma "tradução livre" na primeira menção a termos que escolhemos não traduzir?

**Comment by celsobessa** [comment]
Primero, obrigadão por começar a dicussão. Acho que tem um monte de nuances e discussões subjacentes (e.g. adaptar VS traduzir) e não vamos esgotar aqui.

TL;DR (ou LD;NL ? :-) )

- Adaptar é melhor que traduzir;
- Usar pt-br na maioria das vezes;
- um termo em português vai ser facilmente entendido por quem entende inglês. Já o contrário é menos provável.


Minha posição tem sido traduzir e adaptar o máximo possível para o português brasileiro.

Intuo que o "português brasileiro é uma língua que abraça muito termos emprestados de outros idiomas" acontece por diferentes razões com diferentes idiomas. Por exemplo, a absorção de idiomas de culturas e populações que migraram em massa para o Brasil é diferente de outras como o Francês e o Inglês, que foram idiomas "da elite" por um bom tempo, que depois quando começa a popularizar se ligaram ao pensamento  "o que é de fora" melhor da elite, especialmente abertura do mercado brasileiro no final dos anos 1980 e o crescimento do uso de computadores nos 1990.

Além disto, acessibilidade é um ponto importante para mim e língua e linguagem são fatores que impactam acessibilidade. Evitar jargão é um boa prática de acessibilidade, e eu trato um idioma estrangeiro como jargão. Logo utilizar em pt-br seria mais acessível. 

Na mesma linha, eu presumo que, desconsiderando outros fatores e contexto, para pessoas falantes nativas do português, os termos em português são entendidos de forma fácil e instintiva por mais pessoas que o termo em inglês. Uma pessoa nativa em português e com domínio de inglês vai entender o termo em inglês de forma fácil e instintiva, mas não as pessoas com menos domínio de inglês. Já o termo em português é a linha de base para os dois grupos. Resumindo, o público potencial para português é maior, e se a ideia é tornar o conhecimento mais acessível e um público maior, então o português seria uma melhor escolha

Na minha experiência avaliando curículos, fazendo entrevistas de candidatos, facilitando oficinas e treinamentos, é que quanto mais termos em inglês, maior a percepção de que é necessário saber inglês para fazer x, trabalhar com Y, etc, mesmo que isto não seja a verdade. E aí vem estresse, auto-julgamento, desincentivo para o estudo/trabalho, etc. Então novamente entendo que pode ser alguma barreira.

Tem vários outros pontos, como redução semântica, bem comum no meio tecnológico brasileiro (ligado ao fenômeno do inglês técnico) que acaba limitando não só o português, como próprio inglês (a própria palavra semântica é um exemplo de redução semântica na área de programação web), mas deixemos para outro momento da discussão.

De qualquer forma:

1. o abraço aos termos começou com escolhas de algumas pessoas em circunstâncias, e em teoria nada impediria que uma decisão de abraçar o português brasileiro nestas traduções não tenha efeitos positivos maiores.
2. como disto, há matizes nesta discussão, e não considero os pontos acima argumentos definitivos. Estão aí só para dar andamento na discussão.
3. Me parece que vamos ter que fazer uma escolha e sempre haverão casos em que ela vai se mostrar errada, seja qual for a escolha. Então talvez seja bom também pensar como lidar com isto,  como avaliar, como se recuperar de uma escolha que cause mais dano que benefício (afinal, não é binário), etc.
4. no fim, sou apenas alguém com opinião demais e sabedoria de menos.

PS: O inglês técnico é bom para começar e aprender certas coisas rapidamente, mas só até um ponto e num espaço/escopo bem limitado. Mais jargão que idioma. Mas novamente, opinião demais, sabedoria de menos.

**Reply by contraexemplo** [reply]
💯. Eu considero muito importante fazermos essas escolhas de forma consciente também, em alguns casos, círculos e situações, tenho observado uma mudança no vocabulário no português por influência da língua inglesa (no uso de, por exemplo, "realizar" para dizer "perceber", "codar" para dizer "programar" — mas esse é um processo que _já aconteceu_ com palavras como "resetar" ou "postar"). Há, também, uma linha tênue entre alcançar entendimento através do uso dos termos traduzidos ou os termos originais, ou até mesmo um dinamismo no texto e na fala (por exemplo, usar "notebook" ou "computador portátil"?).

Acho que vamos acabar adotando uma política bem caso-a-caso: _phishing_, por exemplo, é um termo tão recorrente e tão usado até fora da bolha de tecnologistas que tenho o mantido em inglês.

**Reply by celsobessa** [reply]
Só um comentário, não necessariamente um argumento por traduzir. No lugar _phishing_, já usei oralmente em oficinas "este email é uma _isca_", "este é um caso de _isca_", "_isca_ de golpe"/"golpe de _isca_".

**Comment by yamakat** [comment]
tô de acordo com tudo o que já foi posto aqui, reforçando que talvez seja mais interessante mesmo irmos caso a caso, e considerando que uma palavra não deve ser, realmente, uma barreira, e sim uma ponte (no sentido de não decretar deixarmos sempre em ptbr simplesmente pra deixar em ptbr, mas facilitar o entendimento e contextualização de quem lê).

daí, também sou à favor de usarmos traduções livres entre parênteses quando usamos termos emprestados do inglês/deixamos de traduzir pra ptbr.
